pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -1930,10 +1943,6 @@ public State getState() {
         return state;
     }
 
-    public SocketAddress getRemoteAddress() {
```

Comment:
This does the same thing that `clientAddress()`. Should I keep it for backward compat ? It would be dead code... So maybe with a deprecation annotation ?